contrib/syslog-debun

```diff
@@ -456,6 +546,172 @@ debun_extra_debian () {
 	md5sum --quiet -c /var/lib/dpkg/info/syslog-ng*.md5sums && echo Package files are intact
 	echo "list syslog-related packages"
 	dpkg -l |grep syslog > ${tmpdir}/deb.packages
+}
+
+debun_extra_redhat () {
+	printf "\nRedhat specific checks\n"
+	echo "Check package files integrity"
+	rpm -V syslog-ng-premium-edition && echo "syslog-ng-premium-edition: Package files are intact"
+	rpm -V syslog-ng-premium-edition-client && echo "syslog-ng-premium-edition-client: Package files are intact"
+	rpm -V syslog-ng-premium-edition-compact && echo "syslog-ng-premium-edition-compact: Package files are intact"
+	echo "list syslog-related packages"
```

Comment:
I suggest to not copy code, and solve this problem like this:
pkgfound=0
for pkg in pkgname1 pkgname2 ... 
do
  if rpm -q $pkg ;
    pkgfound=$[pkgfound+1]
    if rpm -V $pkg;
     printf "%s: Package files are intact\n" "$pkg"
   else
     printf "%s: Package files are borked\n" "$pkg"
    fi
  fi
done
if [ $pkgfound -eq 0 ]; then
echo "I haven't found any syslog package on your system"
fi